anomalyze.go

```diff
@@ -139,6 +139,18 @@ func (a *Anomalyzer) Push(x float64) float64 {
 	return a.Eval()
 }
 
+// WARNING: Mixing Push() and PushFixed() will result in failure!
```

Comment:
woo! I should have caught that.. I think I fixed all this stuff on a separate branch but it's upstream to far to cherry pick cleanly..